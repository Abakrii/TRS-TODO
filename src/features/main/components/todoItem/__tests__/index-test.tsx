import React from 'react';
import renderer from 'react-test-renderer';
import TodoItem from '../';

test('renders correctly', () => {
    const tree = renderer.create(<TodoItem
        savedTodoKey={[{ id: "1", title: "totdo title" }]}
        onSave={() => console.log('on save is working')}
        item={{ id: "1", title: "text item text", }}
        onDelete={() => console.log('on delete')}
    />).toJSON();
    expect(tree).toMatchSnapshot();
});


