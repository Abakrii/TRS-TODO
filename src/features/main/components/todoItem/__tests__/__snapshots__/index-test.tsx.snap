// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "borderColor": "#BBB",
      "borderRadius": 10,
      "borderStyle": "dashed",
      "borderWidth": 1,
      "flex": 1,
      "flexDirection": "row",
      "justifyContent": "space-between",
      "marginTop": 16,
      "padding": 16,
    }
  }
>
  <TextInput
    allowFontScaling={true}
    rejectResponderTermination={true}
    style={
      Object {
        "width": "70%",
      }
    }
    underlineColorAndroid="transparent"
  >
    text item text
  </TextInput>
  <View
    style={
      Object {
        "alignItems": "center",
        "flexDirection": "row",
      }
    }
  >
    <View
      accessibilityRole="button"
      accessibilityState={Object {}}
      accessible={true}
      focusable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "opacity": 1,
        }
      }
    >
      <View
        style={
          Array [
            Object {},
          ]
        }
      >
        <Text
          style={
            Array [
              Object {
                "color": "#007AFF",
                "fontSize": 18,
                "margin": 8,
                "textAlign": "center",
              },
              Object {
                "color": "red",
              },
            ]
          }
        >
          codes.common.delete
        </Text>
      </View>
    </View>
  </View>
  <Text>
    codes.common.done
  </Text>
</View>
`;
